src/bci_build/package/nginx/README.md.j2

```diff
@@ -0,0 +1,92 @@
+# {{ image.pretty_name }} Container Image
+
+{% include 'badges.j2' %}
+
+## Description
+
+Nginx (pronounced "engine-x") is an open-source reverse proxy server for HTTP, HTTPS, SMTP, POP3, and IMAP protocols, as well as a load balancer, HTTP cache, and a web server (origin server).
+
+## Usage
+
+By default, the image launches Nginx with the same configuration that comes with SUSE Linux Enterprise Server.
+
+```ShellSession
+$ podman run -it --rm -p 8080:80 {{ image.pretty_reference }}
+```
+
+Or:
+
+```ShellSession
+$ podman run -it --rm -p 8080:80 -v /path/to/html/:/srv/www/htdocs/:Z {{ image.pretty_reference }}
+```
+
+**Note:** The directory `/srv/www/htdocs/` is the root directory used by the default server. Additional servers can use any other path.
+
+You can access the hosted pages by opening http://localhost:8080 or http://host-ip:8080 in your browser.
+
+## Using templates
+
+Out of the box, nginx doesn't support environment variables inside configuration blocks. This image includes a script that can extract environment variables before nginx starts to create configuration files.
+
+The script reads template files in `/etc/nginx/templates/*.template` and outputs the result of executing `envsubst` to the directory `/etc/nginx/conf.d/`.
+
+If you create a file named `/etc/nginx/templates/default.conf.template`, which contains a variable reference like this:
+
+```
+listen       ${NGINX_PORT};
+```
+
+Setting the environment variable `NGINX_PORT` to `80` will trigger the script to render the template above to `/etc/nginx/conf.d/default.conf` as follows:
+
+```
+listen       80;
+```
```

Comment:
Here's my suggestion.:

For example, if you want Nginx to use port 80, create a file named `/etc/nginx/templates/default.conf.template` containing the following variable definition:

```
listen       ${NGINX_PORT};
```

This makes the script to render the template above to `/etc/nginx/conf.d/default.conf` as follows:

```
listen       80;
```

Reply:
Yes, better indeed.